lib/inspec/shell.rb

```diff
@@ -36,8 +36,13 @@ def configure_pry # rubocop:disable Metrics/AbcSize
       end
 
       # configure pry shell prompt
-      Pry.config.prompt_name = "inspec"
-      Pry.prompt = [proc { "#{readline_ignore("\e[1m\e[32m")}#{Pry.config.prompt_name}> #{readline_ignore("\e[0m")}" }]
+      Pry::Prompt.add(
```

Comment:
The previous line was causing a deprecation warning. Does this new line work with the lowest `">= 0.8"` constraint for method_source ?

Reply:
Nice catch. It appears the `#add` API was added in `pry` 0.13.1 https://github.com/pry/pry/commit/3da930f908ff73f7638bd0471c25d47c28320df3

Reply:
And we now require pry 0.13, cool